R/meta_utils.R

```diff
@@ -152,9 +159,33 @@ get_meta_subset.data.frame <- function(x, row_id, column,
     stop("If querying metadata without a tplyr_table, a target must be provided", call.=FALSE)
   }
 
-  target %>%
+  if (length(list(...)) > 0) {
+    warning("Extra arguments were provided to get_meta_subset() that will not be used.")
+  }
```

Comment:
Is there a reason we allowed for the ellipsis in the first place here? Would it be easier to remove it and let R report an argument error?

Reply:
The problem is there is no error - This is reported here: https://github.com/atorus-research/Tplyr/issues/102

If something truly passes through the ellipsis, it may still process fine and just have no effect, but not produce what you expected. The ellipsis itself is necessary though because of the S3 method. So I chose to add a warning to report it.